### DISCUSSION TITLE
Wrong issue in changelog

### DISCUSSION THREAD

**Discussion by lud** [klarna/brod#494] [General]
https://github.com/kafka4beam/brod/blob/2253f675c6c2fb0527366a43546eb45d5be086ff/changelog.md?plain=1#L8

Hello, I was trying to understand the changes to brod before upgrading my dependencies. It looks like the referenced issue should be https://github.com/kafka4beam/kafka_protocol/issues/89 .

Thank you.

**Comment by zmstone** [comment]
Hi @lud 
It's https://github.com/kafka4beam/kafka_protocol/pull/92

**Reply by zmstone** [reply]
fixed the changelog: https://github.com/kafka4beam/brod/commit/a0b16765b8b85d6e8fd47c439d4f1f7c6ef878b1

**Reply by lud** [reply]
Cool thanks !
